Azure/sonic-buildimage
# [Wistron] Add 6512-32r platform support

#### Why I did it

Add 6512-32r support for wistron platform

#### How I did it

Add code in platform and device folder to support wistron platform

#### How to verify it

Run on wistron platform and run command to verify

Signed-off-by: RogerChang <Roger_Chang@wistron.com>



#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
N/A

#### Link to config_db schema for YANG module changes
N/A

#### A picture of a cute animal (not mandatory but encouraged)
